citation.bib

```diff
@@ -0,0 +1,6 @@
+@article{narayanan2024aviary,
```

Comment:
Can you bring this to match the GitHub `CITATION.cff`

<img width="364" alt="image" src="https://github.com/user-attachments/assets/3de15ae9-5f6b-41dc-b702-7136aaac7c52" />

Also, can you add a trailing newline so that red symbol goes away

<img width="67" alt="image" src="https://github.com/user-attachments/assets/ecd05095-a9e5-4395-8c6e-be5af26989fd" />

Reply:
I don't feel strongly here, but given we have `CITATION.cff` I don't think we need `citation.bib`